src/applications/facility-locator/actions/index.js

```diff
@@ -240,6 +243,18 @@ export const genBBoxFromAddress = query => {
         const zipCode = zip.text || features[0].place_name;
         const featureBox = features[0].box;
 
+        dispatch({
+          type: GEOCODE_COMPLETE,
+          payload: query.usePredictiveGeolocation
+            ? features.map(feature => ({
+                placeName: feature.place_name,
+                placeType: feature.place_type,
```

Comment:
Place name is an array you might want to join it here - `place_type.join(' ')` if need